core/src/main/scala/pureconfig/ConvertHelpers.scala

```diff
@@ -94,36 +102,68 @@ trait ConvertHelpers {
     override def to(t: T): ConfigValue = ConfigValueFactory.fromAnyRef(t)
   }
 
-  def fromStringReader[T](fromF: String => Option[ConfigValueLocation] => Either[ConfigReaderFailure, T]): ConfigConvert[T] = new ConfigConvert[T] {
+  def fromStringReader[T](fromF: String => Option[ConfigValueLocation] => Either[ConfigReaderFailure, T]): ConfigReader[T] = new ConfigReader[T] {
```

Comment:
Do you think we should move reader/writer helpers in the `Reader`/`Writer` companion object? I think it could be nice to have `ConfigReader.fromString` instead of `ConverterHelper.fromStringReader`. You don't need the `Reader` at the end of `fromStringReader` if the object in which the method is defined is a `Reader` (I hope I explained myself). What do you think?